### ISSUE TITLE
Zoom behavior when list items have same name

### ISSUE THREAD

**Issue by Polarisation** [blinry/almightree#3]
In this case, zooming will highlight all items with that name, rather than the specific item that was selected.

It's easy to reproduce this by adding this line twice to the demo HTML:
`<li>This element is duplicated in both first-level items</li>`

**Comment by blinry** [comment]
Thanks, I will look into that. So far, this was expected and intended behaviour, but I agree its confusing. I guess we could include the parents of the selected item in the resulting search term, like "first-level-iteem/this-element-is-duplicated...".

**Comment by Polarisation** [comment]
Yeah that would do it. Or alternatively number the instances of that term, so the search term could be something like "duplicated-element/2". I have no strong preference.
